.gitignore

```diff
@@ -22,6 +22,7 @@ filenametags
 *.coverprofile
 
 coverage.txt
+coverage.out
```

Comment:
where does this file come from?

Reply:
run `go tool cover -V -func=coverage.txt -o coverage.out ` will create a file which named coverage.out